### DISCUSSION TITLE
How can I get appium session detail with webdriverio

### DISCUSSION THREAD

**Discussion by kidmtt94** [webdriverio/webdriverio#10448] [Q&A]
Hi guys,

I'm using webdriverio 8.3.5.
I try to attach and get session detail information to retrieve pixel `ratio` of **mobile** device.

Here is my code:

<pre><code>const session = await attach(attachOptions);
const detail = await session.getSession();
</code></pre>

But it always return `TypeError: session.getSession is not a function`
As I read, getSession is **Jsonwp Protocol Command**. And the description of this method linked to legacy document. Do we have any method belong to W3C to retrieve session details ?
My main purpose is retrieving the pixel ratio.

Please help me in this case.

**Comment by christian-bromann** [comment]
@kidmtt94 if you like to access the session capabilities returned by the Appium, you can just do `browser.capabilities`, see more: https://webdriver.io/docs/api/browser/

**Reply by kidmtt94** [reply]
Thank you for your answer.
I did but it doesn't work for me.
`browser.capabilities` is always empty object.
![image](https://github.com/webdriverio/webdriverio/assets/12729125/f406bec9-7a1d-4b75-9f7b-123a97f7b38f)

**Note**: Here is my capabilities
```
{
  "appium:app": "F:\\Sample\\test.apk",
  "appium:platformVersion": "13",
  "appium:automationName": "UiAutomator2",
  "platformName": "Android",
  "appium:udid": "emulator-5554",
  "appium:deviceName": "emulator"
}
```

And I just want to retrieve session information like we invoke API `http://127.0.0.1:4723/wd/hub/session/{sessionId}`
Do you have any idea ?
